blocksuite/affine/components/src/rich-text/keymap/format.ts

```diff
@@ -20,8 +20,20 @@ export const textFormatKeymap = (std: BlockStdScope) =>
             const textSelection = selection.find(TextSelection);
             if (!textSelection) return;
 
+            // checks text length
```

Comment:
You should modify the link config instead of the handler.

Reply:
Yes, that's easier